src/rust/engine/src/context.rs

```diff
@@ -257,13 +257,8 @@ impl Core {
       None
     };
     let ignorer =
-      GitignoreStyleExcludes::create_with_gitignore_file(&ignore_patterns, gitignore_file)
-        .map_err(|e| {
-          format!(
-            "Could not parse build ignore inputs {:?}: {:?}",
-            ignore_patterns, e
-          )
-        })?;
+      GitignoreStyleExcludes::create_with_gitignore_file(ignore_patterns, gitignore_file)
+        .map_err(|e| format!("Could not parse build ignore patterns: {:?}", e))?;
```

Comment:
Why not include the `ignore_patterns` in the error anymore? Is it already included in the error message.

Reply:
Yea, the failing glob is propagated from below.